cmd/sso.go

```diff
@@ -25,9 +28,22 @@ var (
 		GroupID: groupManagementAPI,
 		Use:     "sso",
 		Short:   "Manage Single Sign-On (SSO) authentication for projects",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
using `PersistentPreRunE` so that the validity checks are being enforced across all subcommands

Reply:
One caveat about `PersistentPreRunE` is that it doesn't inherit from parent command properly. So you also need sth [like this](https://github.com/supabase/cli/blob/main/cmd/db.go#L134) 
```suggestion
		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
			if err := cmd.Root().PersistentPreRunE(cmd, args); err != nil {
				return err
			}
```